src/test/java/com/amazonaws/encryptionsdk/model/KeyBlobTest.java

```diff
@@ -170,4 +172,51 @@ public void checkKeyLen() {
 
         assertEquals(mockDataKey_.getEncryptedDataKey().length, reconstructedKeyBlob.getEncryptedDataKeyLen());
     }
+
```

Comment:
In addition to the direct method calls, we should have a static test vector for each of these that contains bytes that will trigger the correct error when parsed.

Reply:
How should these vectors be provided? As a byte array literal?

Reply:
We're working on building out more standardized approaches for these, but for the time being, yes, just some byte array literals will do nicely.

Reply:
I added the vectors as functions, but wasn't sure if a field would be preferable. I would be glad to present them in whatever manner would be most useful.